homeassistant/components/sia/alarm_control_panel.py

```diff
@@ -108,6 +109,11 @@ def __init__(
 
         self._attr_state: StateType = None
         self._old_state: StateType = None
+        self._attr_supported_features = (
+            AlarmControlPanelEntityFeature.ARM_AWAY
+            | AlarmControlPanelEntityFeature.ARM_NIGHT
+            | AlarmControlPanelEntityFeature.ARM_CUSTOM_BYPASS
```

Comment:
This entity doesn't implement the corresponding services. The supported features are required for the services, not for the state.

Reply:
You're welcome to open an architecture discussion in our architecture repo about handling supported state for triggers etc.

For now, the best fix is probably to allow all device triggers and conditions in the alarm control panel.

Reply:
Made the fix here. I will create a discussion tomorrow